corehq/apps/case_importer/exceptions.py

```diff
@@ -106,14 +106,6 @@ class DuplicateLocationName(CaseRowError):
     )
 
 
-class InvalidLocation(CaseRowError):
-    title = ugettext_noop('Invalid Location')
-    message = ugettext_lazy(
-        "The location of the case owner needs to be at or below the "
-        "location of the user importing the cases."
-    )
```

Comment:
It seems unfortunate to have to lose this context for a generic "owner ID is invalid".  Seems reasonable to declare out of scope, but it would be nice to push the exception raising into the owner ID validators so we can give more specific feedback.